### ISSUE TITLE
Possible to visualize on District Level?

### ISSUE THREAD

**Issue by FBosler** [mfreeborn/coronavirus-tracker#2]
Hi Michael,

I love your Notebook!

I could not find contact information, which is why I am using an Issue. 

I built a plotly app to visualize the spread throughout Germany (as can be found [here](https://corona.fabianbosler.de/)) - Unfortunately, it crashes quite often now (as I included more dates), which is why I wanted to move from Plotly Express to the PlotlyVersion you use (I feel like having the frames predefined will make it easier to load).

But I could not figure out how to set that up with custom GeoJson.

If you got pointers, hit me up on LinkedIn or [Medium](https://medium.com/@fabianbosler) 

Cheers
Fabian

**Comment by mfreeborn** [comment]
Thank you, sorry for the delayed reply - work's been keeping me busy! Certainly whenever I started either these visualisations, or other visualisations with Plotly, I began with plotly express. However, most of the time I end up dropping down to the plain plotly API (i.e. not express) because it allows for more complex charts to be designed without trying to be too clever and doing it all in one God function. I'm afraid I don't have much experience on the GeoJson front.
